### PR TITLE
Ee

### PR DESCRIPTION
**Pull request by DrewParallel**
Changes for execution environment support and removing groups.

### CODE REVIEW

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    agent_name = ee[EEDefs.AGENT]
+    agents = mclient.list_agents()
+    if agents is None:
+        raise Exception("Could not receive agent list")
+    if len(agents) == 0:
+        raise Exception("No agents are defined in MCenter")
+
+    agent = list(filter(lambda x: x["host"] == agent_name, agents))
+    if len(agent) == 0:
+        raise Exception("Could not find agent {} in MCenteer".format(agent_name))
```
Comment (DrewParallel) [inline reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:72]: typo

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        sys.exit(1)
+
+    ee = ee[0]
+
+    # Deleting fields which are not needed when saving as a file
+    del_if_exists(ee, EEDefs.ID)
+    del_if_exists(ee, EEDefs.TEST_PIPELINE_ID)
+    del_if_exists(ee, EEDefs.TEST_MLAPP_ID)
+
+    # Setting the agent ID to agent name
```
Comment (DrewParallel) [inline reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:39]: Set the agent id from the agent name

File: reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    if agents is None:
+        raise Exception("Could not receive agent list")
+    if len(agents) == 0:
+        raise Exception("No agents are defined in MCenter")
+
+    agent = list(filter(lambda x: x["host"] == agent_name, agents))
+    if len(agent) == 0:
+        raise Exception("Could not find agent {} in MCenteer".format(agent_name))
+    if len(agent) > 1:
+        raise Exception("There are more then one agent with name: {}".format(agent_name))
```
Comment (DrewParallel) [inline reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/execution_environment.py:74]: are --> is
then --> than

**Review by dulcardopm — APPROVED** [review]

**Review by liororama — APPROVED** [review]

**Comment by DrewParallel** [comment]
Pushed to master

**Pull request closed without merging**